.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/manuelschmid/dev/mystudy/src/Components/TitleElement/TitleElement.stories.tsx":"1","/Users/manuelschmid/dev/mystudy/src/Components/TitleElement/TitleElement.tsx":"2"},{"size":190,"mtime":1638179238269,"results":"3","hashOfConfig":"4"},{"size":343,"mtime":1638179238269,"results":"5","hashOfConfig":"4"},{"filePath":"6","messages":"7","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"17owpm0",{"filePath":"8","messages":"9","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/manuelschmid/dev/mystudy/src/Components/TitleElement/TitleElement.stories.tsx",[],"/Users/manuelschmid/dev/mystudy/src/Components/TitleElement/TitleElement.tsx",[]]
```

Comment:
You could at this file to `.gitignore`.

Reply:
I pushed this change to main branch. So it should be solved now.